ui/src/constants.js

```diff
@@ -26,3 +26,5 @@ export const STATUS_UNKNOWN = 'Unknown';
 
 export const SPARSE_LOOP_DEVICE = 'sparseLoopDevice';
 export const RAW_BLOCK_DEVICE = 'rawBlockDevice';
+
+export const SET_SEARCHED_VOLUME = 'SET_SEARCHED_VOLUME';
```

Comment:
You should rename this constant ```SET_SEARCHED_VOLUME``` to something like ```VOLUME_SEARCH_TEXT```since SET_SEARCHED_VOLUME means an action